CONTRIBUTING.md

```diff
@@ -15,47 +15,67 @@ Here are some important resources:
 
 ## Testing
 
- We make sure Travis-CI can build your pull-requests before we can accept your contributions.
-
-## Submitting changes
-
-Please send a [Pull Request to Harvey OS](https://github.com/Harvey-OS/harvey/pull/new/master) with a clear list of what you've done (read more about [pull requests](http://help.github.com/pull-requests/) and [Github Flow](https://guides.github.com/introduction/flow/)).
-
-Some of our contributors like using [hub](http://hub.github.com) for sending pull requests from the command line, an example workflow for contributing to harvey would look like (after you've [installed hub](http://hub.github.com))
-``` 
-$ git clone git@github.com:Harvey-OS/harvey.git
-$ cd harvey
-```
-create a topic branch
-``` 
-$ git checkout -b feature
-  ( making changes ... )
-$ git commit -m "some descriptive thing about what you have done"
-``` 
-It's time to fork the repo!
-``` 
-$ hub fork
-→ (forking repo on GitHub...)
-→ git remote add YOUR_USER git://github.com/YOUR_USER/harvey.git
-```
-push the changes to your new remote
-``` 
-$ git push YOUR_USER feature
-```
-open a pull request for the topic branch you've just pushed
-``` 
-$ hub pull-request
-→ (opens a text editor for your pull request message)
-```
-and someone from the core team will review it. 
-
-When you send a pull request, we will love you forever if you include qemu outputs and more tests. We can always use more test coverage. Please follow our coding conventions (below) and make sure all of your commits are atomic (one feature per commit).
-
-Always write a clear log message for your commits. One-line messages are fine for small changes, but bigger changes should look like this:
+We make sure Travis-CI can build your pull-requests before we can accept your contributions.
 
+## Submitting a patch
+
+Harvey uses Github Pull Requests to accept contributions.
+
+1.  Clone the repo `git clone https://github.com/Harvey-OS/harvey.git`. It is also possible to use git instead of https when you have an SSH public key stored in Github. Then you clone the repo with `git clone git@github.com/Harvey-OS/harvey.git`. This makes later submissions easier. For the rest of this manual we assume to use https.
+
+2.  Check out a feature branch you will work on by `git checkout -b feature-name` for instance for issue #70, @keedon selected the branch name `statscrash`.
+
+3.  Make changes
+
+4.  Commit with a descriptive message, like so
+    ```
     $ git commit -m "A brief summary of the commit
     >
     > A paragraph describing what changed and its impact."
+    ```
+    take a look at @keedon's [commit message](https://github.com/keedon/harvey/commit/09fe3a21fa8b42088bc8ad83287928e9e7cc96ef) for the same issue for a good example of a descriptive commit message.
+    You can also use graphical git tools such as `git cola` which make commiting changes easier.
```

Comment:
`git cola` is not a distributed with git and you haven't provided a link for it. However `git gui` is and it works cross platform so would you mind swapping out `git cola` for `git gui`.

you could maybe mention in the more information section [Githubs Desktop Client](https://desktop.github.com/) for osx and windows which supports pull requests.